src/hotspot/share/opto/intrinsicnode.cpp

```diff
@@ -368,3 +370,23 @@ const Type* ExpandBitsNode::Value(PhaseGVN* phase) const {
 
   return bitshuffle_value(src_type, mask_type, Op_ExpandBits, bt);
 }
+
+Node* ScopedValueGetLoadFromCacheNode::scoped_value() const {
+  Node* hits_in_cache = in(1);
+  return hits_in_cache->as_ScopedValueGetHitsInCache()->scoped_value();
+}
+
+IfNode* ScopedValueGetLoadFromCacheNode::iff() const {
+  return in(0)->in(0)->as_If();
```

Comment:
```suggestion
  return in(0)->as_IfTrue()->in(0)->as_If();
```
Would make verification below implied.